contracts/contracts/coordination/IEncryptionAuthorizer.sol

```diff
@@ -2,8 +2,8 @@ pragma solidity ^0.8.0;
 
 interface IEncryptionAuthorizer {
     function isAuthorized(
-        uint32 ritualID,
-        bytes memory evidence, // signature
-        bytes32 digest // signed message hash
+        uint32 ritualId,
+        bytes memory evidence, // supporting evidence for authorization
+        bytes memory ciphertextHeader // data to be signed by authorized
```

Comment:
👏 .

@cygnusv Is the ciphertext header a fixed size? If so, any benefits that we can leverage from that fact like `bytes32` or ...?